src/utils.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * utils.ts
+ * Copyright: Microsoft 2019
+ */
+
+export const assert = (cond: any, message?: string | undefined) => {
```

Comment:
cond: any should be boolean probably
Also let's make message mandatory here - we want to know what the failure was